.eslintrc.json

```diff
@@ -16,7 +16,8 @@
     "@typescript-eslint/no-extra-semi": "off",
     "@typescript-eslint/no-empty-function": "off",
     "simple-import-sort/imports": "error",
-    "simple-import-sort/exports": "error"
+    "simple-import-sort/exports": "error",
+    "@typescript-eslint/no-namespace": ["error", { "allowDeclarations": true }]
```

Comment:
I am not sure if this is still necessary. Worst case we can `// eslint-ignore` in one place